de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/ui/eventhandler/SessionStatusRequestHandler.java

```diff
@@ -82,16 +87,17 @@ private void handleStatusRequest(JID from) {
     }
 
     private String getSessionDescription(ISarosSession session) {
-        String description = "Projects: ";
+        String description = "ReferencePoints: ";
 
-        Set<IProject> projects = session.getProjects();
+        Set<IProject> projects = referencePointManager.getProjects(session
+            .getReferencePoints());
         int i = 0;
         int numOfProjects = projects.size();
 
         for (IProject project : projects) {
             description += project.getName();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a StringBuilder instead. ([squid:S1643](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1643))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=258&issues=AWbe6KHGSgN4uCclR40E&open=AWbe6KHGSgN4uCclR40E)

[ISSUE_KEY]: AWbe6KHGSgN4uCclR40E